config/core/build-configs.yaml

```diff
@@ -274,6 +274,7 @@ fragments:
       - 'CONFIG_PINCTRL_AMD=y'
       - 'CONFIG_NVME_CORE=y'
       - 'CONFIG_BLK_DEV_NVME=y'
+      - 'CONFIG_EXTRA_FIRMWARE="amdgpu/stoney_ce.bin amdgpu/stoney_me.bin amdgpu/stoney_mec.bin amdgpu/stoney_pfp.bin amdgpu/stoney_rlc.bin amdgpu/stoney_sdma.bin amdgpu/stoney_uvd.bin amdgpu/stoney_vce.bin rtl_nic/rtl8153b-2.fw"'
```

Comment:
Wrapping this with one firmware file per line would help with readability and also it will make it easier to maintain if we need to add/remove some in the future.

Reply:
Will do